DESCRIPTION

```diff
@@ -22,12 +22,9 @@ LazyData: true
 ByteCompile: true
 RoxygenNote: 7.1.1
 Depends: 
-    R (>= 3.5.0)
+    R (>= 3.3.0)
```

Comment:
Has this been tested on R 3.3? (Should be easy to do with rstudio.cloud)

Reply:
No, but I'll take your suggestion and do the testing on Cloud. Also, the CI passes with this change so that's promising.

Reply:
Just did the testing on RStudio Cloud with R 3.3.3. Used the Shiny and R Markdown testing files in `tests/manual_tests` and both worked.